### DISCUSSION TITLE
Welcome to VTP-web-client Discussions!

### DISCUSSION THREAD

**Discussion by windoverwater** [TrustTheVote-Project/VTP-web-client#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by windoverwater** [comment]
@stratofax success?

**Reply by stratofax** [reply]
@windoverwater Thanks for setting up the discussion boards

**Comment by trustthevote** [comment]
*Thanks for setting up this Forum, Sandy*.
*Gregory Miller *here; co-founder and COO of the *OSET Institute
<https://osetinstitute.org/>*, the fiscal sponsor of the *TrustTheVote
<https://trustthevote.org/>*® *Project* and big proponent of *VoteTrackr
<https://trustthevote.org/wp-content/uploads/2023/05/VoteTrackr_Chart_72x30_FINAL-.pdf>+*
™
Cheers


Mr. Gregory Miller
Co-Founder, COO
OSET Institute, Inc. <http://www.osetinstitute.org> | TrustTheVote Project
<http://www.trustthevote.org> | Podcast <https://deadmendontvote.org/>
530 Lytton Street, 2nd Floor
Palo Alto, California 94301 USA
Main:   +1.650.600.1450
Mobile: +1.503.703.5150
